cmake/CMakePresets-CI.json

```diff
@@ -33,7 +33,13 @@
           "value": true
         }
       },
+      "warnings": {
+        "dev": true,
+        "deprecated": true,
+        "uninitialized": true
+      },
       "errors": {
+        "dev": true,
```

Comment:
This part is slightly blocked by upstream support to ignore third-party warnings: https://gitlab.kitware.com/cmake/cmake/-/issues/26288